src/keypair.rs

```diff
@@ -53,6 +53,40 @@ impl Keypair {
         let entropy = self.0.secret_to_vec();
         entropy_to_mnemonic(&entropy)
     }
+
+    /// Provides write method for writing the seed to a file. We only support this method for
+    /// ED25519 since we provide this functionality for exporting seed to Solana CLI.
+    pub fn write_unencrypted_seed(&self, writer: &mut dyn io::Write) -> Result {
+        match &self.0 {
+            helium_crypto::Keypair::Ed25519(key) => {
+                // output starts with the 32 byte private key
+                let mut output = key.secret_to_vec();
+                // followed by the 32 byte public key
+                let mut public_key = key.public_key.to_vec();
+                // the first byte (key tag) is removed such that the output
+                // will be compatible with Solana CLI
+                public_key.remove(0);
+                // we now have a 64 byte vector that is Solana compatible
+                output.extend(public_key);
+
+                let mut numbers: String = output
```

Comment:
Why can't you use the json serializer to serialize the vector of bytes?

Reply:
I can, but then I have to read! It's done.